packages/ni-grpc-extensions/pyproject.toml

```diff
@@ -43,8 +43,11 @@ traceloggingdynamic = { version = ">=1.0", platform = "win32" }
 
 [tool.poetry.group.dev.dependencies]
 types-grpcio = ">=1.0"
-# Install traceloggingdynamic on Linux for type checking.
-traceloggingdynamic = { version = ">=1.0", platform = "linux" }
+# Install traceloggingdynamic on Linux and macOS for type checking.
```

Comment:
Now that we run mypy on Windows, we don't need to do this. Also, now we know this won't work for some packages lik pywin32.

Reply:
I will post a fix in a separate PR.